### PR TITLE
Task1 models

### PR DESCRIPTION
**Pull request by Kaid000**

### CODE REVIEW

File: server/RAFwebsite.Models/News.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace RAFwebsite.Models
+{
+    internal class News
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/News.cs:3]: Why New[s]?

File: server/RAFwebsite.Models/News.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        public int news_id { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/News.cs:5]: Simply `Id`. Id is already in the news

File: server/RAFwebsite.Models/News.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        public string main_title { get; set; }
+        public string image { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/News.cs:7]: same

File: server/RAFwebsite.Models/News.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        public string text { get; set; }
+        public Organization organization { get; set; }
+        public Event f_event { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/News.cs:10]: you can use @ to escape keyword usage.
Also, don't use event as a class name. change

File: server/RAFwebsite.Models/User.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+{
+    internal class User
+    {
+        public int user_id { get; set; }
+        public string login { get; set; }
+        public string user_name { get; set; }
+        public string user_surname { get; set; }
+        public string user_email { get; set; }
+        public string user_phone { get; set; }
+        public bool is_admin { get; set; }
```
Comment (BlazeOfSoul) [inline server/RAFwebsite.Models/User.cs:11]: Add Enum class UserRoles with User and Admin

File: server/RAFwebsite.Models/Organization.cs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿namespace RAFwebsite.Models
+{
+    internal class Organization
+    {
+            public string organization_name { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/Organization.cs:5]: Rename to `Name`

File: server/RAFwebsite.Models/Organization.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+            public string leader { get; set; }
+            public string? about { get; set; }
+            public string? logo { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/Organization.cs:8]: Use files from the server

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1,63 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (BlazeOfSoul) [inline .gitignore:1]: Comment for all PR
Create a BaseEntity class and inherit all classes from it

File: server/RAFwebsite.Models/Organization.cs [hunk 9]
```diff
@@ -0,0 +1,13 @@
+﻿namespace RAFwebsite.Models
+{
+    internal class Organization
+    {
+            public string organization_name { get; set; }
+            public string leader { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/Organization.cs:6]: Who is leader?

File: server/RAFwebsite.Models/AboutFaculty.cs [hunk 10]
```diff
@@ -0,0 +1,10 @@
+﻿namespace RAFwebsite.Models
+{
+    internal class AboutFaculty
+    {
+        public string main_title { get; set; }
+        public string? about { get; set; }
+
```
Comment (BlazeOfSoul) [inline server/RAFwebsite.Models/AboutFaculty.cs:7]: Delete pass

File: server/RAFwebsite.Models/Panorama.cs [hunk 11]
```diff
@@ -0,0 +1,7 @@
+﻿namespace RAFwebsite.Models
+{
+    internal class Panorama
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/Panorama.cs:3]: Remove if it's a single Slider (it's called slider, not panorama) on the main page. If it is a something that will be used widely across the website, rename to slider, adjust ot use server files for images

File: server/RAFwebsite.Models/RAFwebsite.Models.csproj [hunk 12]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/RAFwebsite.Models.csproj:5]: Use 8

File: server/RAFwebsite.Models/RAFwebsite.Models.csproj [hunk 13]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/RAFwebsite.Models.csproj:4]: Why exe?

File: server/RAFwebsite.Models/User.cs [hunk 14]
```diff
@@ -0,0 +1,15 @@
+﻿namespace RAFwebsite.Models
+{
+    internal class User
+    {
+        public int user_id { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/User.cs:5]: Simply `Id`

**Review by BlazeOfSoul — CHANGES_REQUESTED** [review]

File: server/RAFwebsite.Models/User.cs [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+        public string login { get; set; }
+        public string user_name { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/User.cs:7]: rename to first name

File: server/RAFwebsite.Models/User.cs [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+        public string user_surname { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/User.cs:8]: rename to last name

File: server/RAFwebsite.Models/User.cs [hunk 17]
```diff
@@ -0,0 +1,15 @@
...
+        public string user_email { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/User.cs:9]: just email

File: server/RAFwebsite.Models/User.cs [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+        public string user_phone { get; set; }
```
Comment (yarmoliq) [inline server/RAFwebsite.Models/User.cs:10]: just `phone`

File: server/RAFwebsite.Models/User.cs (hunk 5 above)
Comment (yarmoliq) [inline server/RAFwebsite.Models/User.cs:11]: Implement Roles, not just a flag

**Comment by yarmoliq** [comment]
Don't use internal. Use CamelCase for properties. Read on how to work properly with images (don't use base64 or whatever it is)

**Review by yarmoliq — CHANGES_REQUESTED** [review]

File: server/RAFwebsite.Models/User.cs [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+        public bool is_admin { get; set; }
+        public string image { get; set; }
```
[the remark this answers is not in the document]
Comment (Kaid000) [reply server/RAFwebsite.Models/User.cs:12]: Is it the class file for images ?

File: server/RAFwebsite.Models/News.cs (hunk 1 above)
Comment (Kaid000) [reply server/RAFwebsite.Models/News.cs:3]: How to name this class correctly?

File: server/RAFwebsite.Models/RAFwebsite.Models.csproj (hunk 13 above)
Comment (Kaid000) [reply server/RAFwebsite.Models/RAFwebsite.Models.csproj:4]: What's would be?

**Pull request closed without merging**